File: corpus/2017q1/_p2q2_ok/_check/D-;0349.java

import java.util.Scanner;
public class AlbadoQuestao2 {

    public static void main(String[] args) {

        Scanner scan= new Scanner (System.in);

        int quant, soma=0;
        int codigo[]= {100, 101, 102, 103, 104, 105};
        String item []= {Pão de Queijo, Bauru, Cachorro Quente, Cheeseburguer, Suco, Refrigerante};
        double preco[]= {1.10, 1.30, 1.50, 1.30, 2.00, 1.00};

        ImprimeCodigo (codigo[]);
        ImprimeItem (item[]);
        ImprimePreco (preco[]);

    public static void ImprimeCodigo (int Vet1[]){
    System.out.println ("Código");
    for (int i=0; i<6; i++) {
    System.out.println(Vet1[i]+"");
     }
    }

    public static void ImprimeItem (int Vet2[]){
    System.out.println ("Item");
    for (int i=0; i<6; i++) {
    System.out.println(Vet2[i]+"");
     }
    }

     public static void ImprimePreco (int Vet3[]){
    System.out.println ("Preço");
    for (int i=0; i<6; i++) {
    System.out.println(Vet3[i]+"");
     }
    }

    while (codigo!=0){
    for (int i=0; i<6; i++){
    System.out.println ("Insira o código do item:"+codigo[i]);
    System.out.println ("Insira a quantidade do item"+quant);
    soma=quant*preco
    }
}         
   System.out.println ("Total da Compra:"+soma);

  }

}

File: corpus/py/D-;0349.py
Scanner scan= new Scanner (System.in)
int quant, soma=0
codigo = [100, 101, 102, 103, 104, 105]
item = [Pão de Queijo, Bauru, Cachorro Quente, Cheeseburguer, Suco, Refrigerante]
preco = [1.10, 1.30, 1.50, 1.30, 2.00, 1.00]
ImprimeCodigo (codigo[])
ImprimeItem (item[])
ImprimePreco (preco[])
def ImprimeCodigo(int Vet1[]):
    print ("Código")
    for i=0 in range(0, 6):
        print(Vet1[i]+"")
def ImprimeItem(int Vet2[]):
    print ("Item")
    for i=0 in range(0, 6):
        print(Vet2[i]+"")
def ImprimePreco(int Vet3[]):
    print ("Preço")
    for i=0 in range(0, 6):
        print(Vet3[i]+"")
while codigo!=0:
    for i=0 in range(0, 6):
        print ("Insira o código do item:"+codigo[i])
        print ("Insira a quantidade do item"+quant)
        soma=quant*preco
print ("Total da Compra:"+soma)